src/Tizen.Multimedia.MediaPlayer/Player/Player.Properties.cs

```diff
@@ -377,12 +377,15 @@ PlayerErrorCode IDisplayable<PlayerErrorCode>.ApplyEvasDisplay(DisplayType type,
                 type == DisplayType.Overlay ? PlayerDisplayType.Overlay : PlayerDisplayType.Evas, evasObject);
         }
 
-        PlayerErrorCode IDisplayable<PlayerErrorCode>.ApplyEcoreWindow(IntPtr windowHandle)
+        PlayerErrorCode IDisplayable<PlayerErrorCode>.ApplyEcoreWindow(IntPtr windowHandle, NUI.Rectangle rect)
         {
             Debug.Assert(IsDisposed == false);
 
+            int width = rect.Width <= 0 ? 1920 : rect.Width;
+            int height = rect.Height <= 0 ? 1080 : rect.Height;
```

Comment:
If you rollback Interop.Display.cs, you don't need to set these values.
I think if Width, Height is 0, it's error case. What do you think about it?

Reply:
Okay. I'll fix it.